.mailmap

```diff
@@ -535,6 +535,7 @@ Ethan Ward <etkewa@gmail.com> eward <eward@sunbelt-medical.com>
 Eva Charlotte Mayer <eva-charlotte.mayer@posteo.de>
 Eva Tiwari <eva.tiwari@gmail.com> Eva <eva.tiwari@gmail.com>
 Evandro <15084103+evbernardes@users.noreply.github.com>
+Evandro Bernardes <evbernardes@gmail.com> Evandro Bernardes <s.evandro@hotmail.com>
 Evandro Bernardes <evbernardes@gmail.com> evbernardes <s.evandro@hotmail.com>
```

Comment:
Is `Evandro <15084103+evbernardes@users.noreply.github.com>` the same person as the other entries?

If so then the first of these lines should be changed to:
```
Evandro Bernardes <evbernardes@gmail.com> Evandro <15084103+evbernardes@users.noreply.github.com>
```
Currently this looks like it's referring to two different authors hence the authors test shows:
```
The following authors will be added to the AUTHORS file at the time of
the next SymPy release.
...
	Evandro <15084103+evbernardes@users.noreply.github.com>
	Evandro Bernardes <evbernardes@gmail.com>
```

Reply:
Oh my, I keep getting this wrong. Thanks again!